packages/rocketchat-livestream/client/views/liveStreamTab.js

```diff
@@ -0,0 +1,65 @@
+import toastr from 'toastr';
+
+Template.liveStreamTab.helpers({
+	streamingSource() {
+		const streamingOptions = Template.instance().streamingOptions.get();
+		if (streamingOptions == null) {
+			return '';
+		} else {
+			const parsedUrl = streamingOptions.url.match(/(http:|https:|)\/\/(clips.|player.|www.)?(twitch\.tv|vimeo\.com|youtu(be\.com|\.be|be\.googleapis\.com))\/(video\/|embed\/|watch\?v=|v\/|embed\?clip=)?([A-Za-z0-9._%-]*)(\&\S+)?/);
```

Comment:
🤔

Reply:
@ggazzo do you need an explanation on how the regular expression works?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match

"If the string matches the expression, it will return an Array containing the entire matched string as the first element, followed by any results captured in parentheses. If there were no matches, null is returned."

you have to  test if `parsedUrl !== null`